README.md

```diff
@@ -162,7 +147,7 @@ public class HomeController : Controller
 ```
 
 ### Controllers and Actions
-MVC controller and actions can require that a given feature, or one of any list of features, be enabled in order to execute. This can be done by using a `FeatureGateAttribute`, which can be found in the `Microsoft.FeatureManagement.Mvc` namespace. 
+MVC controller and actions can require that a given feature flag, or one of any list of feature flags, be enabled in order to execute. This can be done by using a `FeatureGateAttribute`, which can be found in the `Microsoft.FeatureManagement.Mvc` namespace. 
 
 ``` C#
 [FeatureGate(MyFeatureFlags.FeatureX)]

```

Comment:
I think you moved away from using enum (and I like it). The code should be updated to use string.